source/_posts/2022-08-03-release-20228.markdown

```diff
@@ -0,0 +1,560 @@
+providing implementation feedback, and his excellent work on the
+[ble_monitor] custom integration, which was an inspiration for the
+Bluetooth integration. Lastly, a shout out to [@Jc2k] for implementing the
+[Xiaomi BLE] integration that greatly helped with testing. Thanks guys! 🙏
+
+[@bdraco]: https://github.com/bdraco
+[@Ernst79]: https://github.com/Ernst79
+[@Jc2k]: https://github.com/Jc2k
+[ble_monitor]: https://github.com/custom-components/ble_monitor
+[Bluetooth integration]: /integrations/bluetooth
+[Govee]: /integrations/govee_ble
+[INKBIRD]: /integrations/inkbird
+[Moat]: /integrations/moat
+[SensorPush]: /integrations/sensorpush
+[supported Bluetooth adapter]: /integrations/bluetooth#installing-a-usb-bluetooth-adapter
+[SwitchBot]: /integrations/switchbot
+[Xiaomi BLE]: /integrations/xiaomi_ble
+
+## HomeKit Bluetooth support
+
+To continue the story on Bluetooth, the [HomeKit Controller] now also
+supports Bluetooth and leverages the above-mentioned new Bluetooth
+integration 🎉
+
+In case you didn't know, the [HomeKit Controller] integration allows you
+to connect to HomeKit-capable devices and integrate them directly into
+Home Assistant. Allowing you to monitor and control these devices,
+they support push-based updates and, above all: Are entirely local!
+
+You don't need to own or have an Apple phone or computer to use this!
+
+Supporting HomeKit Bluetooth-capable devices is fantastic! It makes a lot of
+devices out there compatible with Home Assistant that have no other way of
+integrating. There are lots of door locks, wall plugs, and other smart devices
+that you can use now.
+
+Thanks for the work on this [@Jc2k] and [@bdraco]!
+
+[@bdraco]: https://github.com/bdraco
+[@Jc2k]: https://github.com/Jc2k
+[HomeKit Controller]: /integrations/homekit_controller
+
+## Finding the integration supporting your device
+
+Some integrations work with multiple different brands. This can happen because,
+for example, those devices share a communication protocol, are white-labeled
+and sold under numerous brand names, or may carry different brand names
+depending on where in the world they are sold.
+
+For example, if you have a Luxaflex cover, you can actually use it using the
+Hunter Douglas PowerView integration. And this is not the only example;
+there are many more! The problem is, how would one know that?
+
+To make setting up things in Home Assistant easier, integrations can now
+support multiple brands! So, if you now add an integration in Home Assistant
+and search for Luxaflex, you will find one!
+
+<img class="no-shadow" src='/images/blog/2022-08/supported-brands.png' alt='Screenshot demonstrating the supported brands; it is now to search for a brand that is supported by another integration'>
+
+When you add a new integration that is supported by another integration,
+Home Assistant will tell you which integration provides the support and
+guides you to set up that integration.
+
+## Better maps colors
```

Comment:
Do we normally talk about maps in plural? The integration is called `map` right? Maybe consider changing it to say "map colors" and "the map integration"?

Reply:
Right. It is not just the integration, but also cards (any map shown anywhere actually). I've adjusted the wording a little.